simulator/Makefile

```diff
@@ -13,6 +13,7 @@ lint:
 
 .PHONY: format
 format:
+	find . -name "*.go" | grep -v "vendor" | grep -v "submodule" | xargs gci write --Section Standard --Section Default --Section "Prefix(sigs.k8s.io/kube-scheduler-simulator/simulator)"
```

Comment:
gci's autofix feature is disabled in this current golangci-lint version. So, manually run it on format.